src/models/NeverminedOptions.ts

```diff
@@ -1,39 +1,46 @@
 import { ethers } from 'ethers'
-import { ZeroAddress } from '../utils'
+import { ZeroAddress } from '../utils/ConversionTypeHelpers'
 import { LogLevel } from '../utils/Logger'
 import { AaveConfig } from './AaveConfig'
 export { LogLevel } from '../utils/Logger'
 
-export class Config {
+export class NeverminedOptions {
+
     /**
-     * MarketPlace URL.
+     * Ethereum Web3 Provider URL. This Url allows the SDK to connect to a blockchain.
+     * This provider depends on the network where you want to connect (Ethereum Mainnet, Rinkeby, Polygon Matic, etc).
+     * It's possible to use a public available provider or a private/paid one like Infura, Alchemy, etc.
      */
-    public marketplaceUri: string
-
+    public web3ProviderUri?: string
     /**
-     * Marketplace auth token.
+     * URL to the MarketPlace API use to record some Nevermined assets metadata. In a Nevermined network there could be
+     * multiple Marketplace API instances. Each of them typically records the metadata of a specific domain/vertical deployed on a Nevermined network.
+     * @see https://docs.nevermined.io/docs/architecture/marketplace-api/
      */
-    public marketplaceAuthToken?: string
+    public marketplaceUri: string
 
     /**
-     * Node URL.
+     * URL to an existing Nevermined Node. A Nevermined network can have multiple running Nodes.
+     * Each of them facilitate the access control and the data and inftrastructure interaction with web2 elements.
+     * @see https://docs.nevermined.io/docs/architecture/node/
      */
     public neverminedNodeUri: string
 
     /**
-     * Faucet URL.
+     * The public address of the Nevermined Node.
+     * @see {@link neverminedNodeUri}
      */
-    public faucetUri?: string
+     public neverminedNodeAddress?: string
 
     /**
-     * Address of the Node.
+     * Marketplace auth token.
      */
-    public neverminedNodeAddress?: string
+    public marketplaceAuthToken?: string
```

Comment:
we need to get rid of this

Reply:
I think the `marketplaceAuthToken` is still necessary right?